### PR TITLE
MWPW-141916: Replace decoreateMarqueeColumns with style

### PR DESCRIPTION
**Pull request by rl-adobe**
Remove decorateMarqueeColumns function from the utils.js.
Instead complete the stylesheet of the columns element with detailed css selector.

Resolves: [MWPW-191916](https://jira.corp.adobe.com/browse/MWPW-191916)

Test URLs:
- Before: https://main--express--adobecom.hlx.page/express/
- After: https://MWPW-191916-decorateMarqueeColumns--express--adobecom.hlx.page/express/?martech=off

### CODE REVIEW

File: express/styles/styles.css [hunk 1]
```diff
@@ -875,7 +875,7 @@ main .block[data-block-status='initialized'] {
   visibility: hidden;
 }
 
-main .columns.columns-marquee {
+main .section:first-of-type .columns:first-of-type {
```
Comment (vhargrave) [inline express/styles/styles.css:878]: and this one :)

**Review by qiyundai — APPROVED** [review]

File: express/blocks/columns/columns.css [hunk 2]
```diff
@@ -1,4 +1,4 @@
-main .columns.columns-marquee {
+main .section:first-of-type .columns:first-of-type {
```
Comment (vhargrave) [inline express/blocks/columns/columns.css:1]: we can probably get rid of this style

**Review by vhargrave — CHANGES_REQUESTED** [review]
it seems a bit strange to be setting min height and then unsetting it again. Can you see if we can fully remove?

File: express/blocks/columns/columns.css (hunk 2 above)
Comment (rl-adobe) [reply express/blocks/columns/columns.css:1]: From the stage, I see the style in columns.css overrides the one in styles.css.

Let me remove the style from the styles.css.

**Comment by vhargrave** [comment]
I just tested this and the experience on the home page is completely broken for example. Can you take a look? 
https://mwpw-191916-decoratemarqueecolumns--express--adobecom.hlx.page/express/?martech=off

**Review by vhargrave — CHANGES_REQUESTED** [review]
see comment

**Comment by rl-adobe** [comment]
> see comment

It seems because of the sync problem with the stage banch.
Working now.

**Review by vhargrave — APPROVED** [review]

**Pull request merged**